src/components/cards/cards.comp.tsx

```diff
@@ -34,6 +34,10 @@ export const Cards = ({ items, fields, callbacks, customActions, customLabels, p
   };
 
   function renderRow(origField: IConfigDisplayField, value: any) {
+    if (origField.type === 'boolean') {
+        value = value ? true : false;
```

Comment:
Just in case, maybe it also would be nice to check if value  !== ‘false’? WDYT?

Reply:
I don't get your point here, sry. If value is already a boolean value, the check re-assigns it to value (and thus does not change anything). In any other case, `value` is assigned the boolean representation of the original value.

Reply:
Could you please rephrase your question?

Reply:
Well, you check if the field’s type in the configuration is ‘boolean’, you’re not checking if the ‘typeof’ of the value itself is boolean. Therefore, just thought it would be nice to make sure the value isn’t set to a string which equal to ‘false’ due to a passing issue or something (I have seen that before).

Reply:
The missing type check is on purpose there. That is actually the key point of this change: this allows any non-null value to be displayed as a boolean `true`. Without the change, the interface displays `[object]` for any non-null object instead of `true`.

But to be honest: I am not too deep into javascript, though, so if you deem this thinking not appropriate I am open to any suggestions/tips.

Reply:
Ok. Approving this change for now. Let’s see if there’s a use case where a ‘false’ string should also be considered as false.

Reply:
There is actually no reason that the string `"false"` should be considered as a boolean false - even in JavaScript. But I have a somewhat strong opionon on that, so no worries :)